### ISSUE TITLE
Consider adding aria-live="polite" for infinite scroller.

### ISSUE THREAD

**Issue by liamnewmarch** [GoogleChrome/ui-element-samples#37]
Hello!

First of all great work with this repo, I really enjoy exploring these samples and watching the accompanying videos. Keep up the excellent work!

I love the idea of reusing DOM elements in the infinite scroller to boost performance. It did occur to me however that this might be incompatible with screen readers.

Ideally, the scroll container should have an [`aria-live="polite"` attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions). How to trigger new content is open for debate, but maybe a visually hidden button would do the trick?

**Comment by surma** [comment]
cc @robdodson Would appreciate your input on this :)

**Comment by robdodson** [comment]
My understanding is that aria live regions are typically used for alerts and may not be the best fit for this use case.

My hunch is that the entire list should be treated as a single tab stop (so users can easily get in and out) and pressing the up or down arrow keys would move to the next item.

Then the most appropriate role would probably be `feed`. [More documentation on role=feed here](https://rawgit.com/w3c/aria/action-2086/aria/aria.html#feed). Note that since `feed` is ARIA 1.1 it may not be well supported by assistive technology but I do believe it is the semantically correct thing to do

**Comment by liamnewmarch** [comment]
Ah, that makes more sense. Would a pull request be welcome?

Thanks chaps!

**Comment by robdodson** [comment]
@liamnewmarch yeah for sure :)
